src/compas/rpc/proxy.py

```diff
@@ -37,28 +34,42 @@ class Proxy(object):
 
     Parameters
     ----------
-    package : :obj:`str`:, optional
+    package : str, optional
         The base package for the requested functionality.
-        Default is `None`, in which case a full path to function calls should be provided.
-    python : :obj:`str`:, optional
+        Default is None, in which case a full path to function calls should be provided.
+    python : str, optional
         The python executable that should be used to execute the code.
-        Default is ``'pythonw'``.
-    url : :obj:`str`:, optional
+    url : str, optional
         The server address.
-        Default is ``'http://127.0.0.1'``.
-    port : :obj:`int`:, optional
+    port : int, optional
         The port number on the remote server.
-        Default is ``1753``.
-    service : :obj:`str`:, package name to start server.
+    service : str, optional
+        Package name to start server.
         Default is ``'compas.rpc.services.default'``.
-    max_conn_attempts: :obj:`int`, optional
+    max_conn_attempts: int, optional
         Amount of attempts to connect to RPC server before time out.
-    autoreload : :obj:`bool`, ``True`` to automatically reload the proxied package if changes are detected.
+    autoreload : bool, optional
+        If True, automatically reload the proxied package if changes are detected.
         This is particularly useful during development. The server will monitor changes to the files
         that were loaded as a result of accessing the specified `package` and if any change is detected,
         it will unload the module, so that the next invocation uses a fresh version.
-    capture_output : :obj:`bool`, ``True`` to capture the stdout/stderr output of the remote process, otherwise ``False``.
-        In general, ``capture_output`` should be ``True`` when using a ``pythonw`` as executable (default).
+    capture_output : bool, optional
+        If True, capture the stdout/stderr output of the remote process.
+        In general, `capture_output` should be True when using a `pythonw` as executable (default).
```

Comment:
I am not convinced about going to non-monospaced for these things.

```suggestion
        In general, ``capture_output`` should be True when using a ``pythonw`` as executable (default).
```

Reply:
it is based on the numpydoc style guide, but we can change that if you like

Reply:
Not sure... I like the style of monospaced font a bit better, but if numpy does it this way, so be it :P
![image](https://user-images.githubusercontent.com/933277/148549568-1866539c-7594-4b69-a651-d0c4043856f5.png)